{"Modified":"2020-11-30T17:54:00","Published":"2020-11-12T18:15:00","access":{"authentication":"NONE","complexity":"LOW","vector":"LOCAL"},"assigner":"cve@mitre.org","capec":[],"cvss":2.1,"cvss-time":"2020-11-30T17:54:00","cvss-vector":"AV:L/AC:L/Au:N/C:N/I:N/A:P","cwe":"CWE-755","id":"CVE-2020-8767","impact":{"availability":"PARTIAL","confidentiality":"NONE","integrity":"NONE"},"last-modified":"2020-11-30T17:54:00","references":["https://www.intel.com/content/www/us/en/security-center/advisory/intel-sa-00400"],"refmap":{"misc":["https://www.intel.com/content/www/us/en/security-center/advisory/intel-sa-00400"]},"summary":"Uncaught exception in the Intel(R) 50GbE IP Core for Intel(R) Quartus Prime before version 20.2 may allow an authenticated user to potentially enable denial of service via local access.","vulnerable_configuration":[{"id":"cpe:2.3:a:intel:quartus_prime:-:*:*:*:-:*:*:*","title":"cpe:2.3:a:intel:quartus_prime:-:*:*:*:-:*:*:*"}],"vulnerable_configuration_cpe_2_2":[],"vulnerable_product":["cpe:2.3:a:intel:quartus_prime:-:*:*:*:-:*:*:*"]}